static/app/views/performance/newTraceDetails/traceDrawer/details/span/sections/table/rows/keys.tsx

```diff
@@ -0,0 +1,146 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {
+  rawSpanKeys,
+  type RawSpanType,
+} from 'sentry/components/events/interfaces/spans/types';
+import {
+  getSpanSubTimings,
+  isHiddenDataKey,
+  type SubTimingInfo,
+} from 'sentry/components/events/interfaces/spans/utils';
+import {OpsDot} from 'sentry/components/events/opsBreakdown';
+import FileSize from 'sentry/components/fileSize';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {space} from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import type {
+  TraceTree,
+  TraceTreeNode,
+} from 'sentry/views/performance/newTraceDetails/traceModels/traceTree';
+import {GeneralSpanDetailsValue} from 'sentry/views/performance/traceDetails/newTraceDetailsValueRenderer';
+import {getPerformanceDuration} from 'sentry/views/performance/utils/getPerformanceDuration';
+
+import {TraceDrawerComponents} from '../../../../styles';
+
+const SIZE_DATA_KEYS = [
+  'Encoded Body Size',
+  'Decoded Body Size',
+  'Transfer Size',
+  'http.request_content_length',
+  'http.response_content_length',
+  'http.decoded_response_content_length',
+  'http.response_transfer_size',
+];
+
+function partitionSizes(data: RawSpanType['data']): {
+  nonSizeKeys: {[key: string]: unknown};
+  sizeKeys: {[key: string]: number};
+} {
+  const sizeKeys = SIZE_DATA_KEYS.reduce((keys, key) => {
+    if (data.hasOwnProperty(key) && defined(data[key])) {
+      try {
+        keys[key] = parseInt(data[key], 10);
```

Comment:
I dont know if the server sends ints or floats here, but I would `parseFloat` so we accurately reflect the data being sent to us